pandas/indexes/base.py

```diff
@@ -2155,11 +2155,22 @@ def groupby(self, to_groupby):
         -------
         groups : dict
             {group name -> group labels}
-
         """
         return self._groupby(self.values, _values_from_object(to_groupby))
 
     def map(self, mapper):
+        """
+        Apply mapper function to its values.
+
+        Parameters
+        ----------
+        mapper : callable
```

Comment:
hmm, so it seems that ``Series.map`` and ``Index.map/Categorical.map`` have different signatures. (first does not accept a callable, only dict-like, second ONLY a callable, and no ``na_action`` arg).